Examples/RPC/swift/Sources/main.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Dispatch
+import SwiftProtobuf
+
+guard let GoogleAPIKey = ProcessInfo.processInfo.environment["GOOGLE_API_KEY"] else {
+    print("Please set the GOOGLE_API_KEY environment variable.")
+    exit(-1)
+}
+
+var document = Google_Cloud_Language_V1_Document()
```

Comment:
This is obviously much easier to read (yay Swift) but my general feedback on the Rust example applies here. I think this would be easier to grok if it went fully "outside-in" or "inside-out"; mixing them is cognitively more challenging.

Reply:
Thanks, let's discuss outside the PR - as with the Rust example, I've added a few blank lines to hopefully make the structure clearer.